backend/tournesol/views/exports.py

```diff
@@ -165,3 +187,35 @@ def get(self, request, *args, **kwargs):
             )
         )
         return response
+
+
+class ExportPublicAllView(APIView):
+    """Export all the public data in a .zip file."""
+    throttle_scope = "api_export_comparisons"
+    permission_classes = [AllowAny]
+
+    @extend_schema(
```

Comment:
Let's define the same cache rule as the comparisons .csv:

```suggestion

    @method_decorator(cache_page_no_i18n(60 * 10))  # 10 minutes cache
    @extend_schema(
```

Reply:
Done in 3f8d589e1f1b508ce9127e4a8c9635224abd9fb9.